### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by rhmdnd**

### CODE REVIEW

File: bundle/manifests/compliance-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1293,13 +1293,7 @@ spec:
                 - mountPath: /var/run/secrets/serving-cert
                   name: serving-cert
                   readOnly: true
-              nodeSelector:
-                node-role.kubernetes.io/master: ""
```
Comment (Vincent056) [inline bundle/manifests/compliance-operator.clusterserviceversion.yaml:1297]: Sorry I think I missed this, is this an expected behavior?

Comment (rhmdnd) [reply bundle/manifests/compliance-operator.clusterserviceversion.yaml:1297]: I wasn't expecting this to change. Was this due to the patch for adding hypershift deployment support?

Comment (jhrozek) [reply bundle/manifests/compliance-operator.clusterserviceversion.yaml:1297]: I tried to fix this with https://github.com/ComplianceAsCode/compliance-operator/pull/346 btw if we merged https://github.com/ComplianceAsCode/compliance-operator/pull/303 we would probably be alerted of changes like these sooner.

Comment (rhmdnd) [reply bundle/manifests/compliance-operator.clusterserviceversion.yaml:1297]: Even though #303  is still in flight, I'm going to rebase this change so we can pickup https://github.com/ComplianceAsCode/compliance-operator/pull/346

**Pull request closed without merging**